.policy.yml

```diff
@@ -18,13 +18,15 @@ approval_rules:
   - name: one admin has approved (PR contributors not allowed)
     options:
       allow_contributor: false
+      invalidate_on_push: true
     requires:
       count: 1
       permissions: ["admin", "maintain"]
 
   - name: two admins have approved
     options:
       allow_contributor: true
+      invalidate_on_push: true
```

Comment:
note to future reviewers: we intentionally do not want `invalidate_on_push` to be set for this repo